examples/app/gapminder/data.py

```diff
@@ -2,7 +2,8 @@
 
 
 def process_data():
-    from bokeh.sampledata.gapminder import fertility, life_expectancy, population, regions
+    from bokeh.sampledata.gapminder import (fertility, life_expectancy,
+                                            population, regions)
```

Comment:
Why are we using different formatting in examples? This is also the worst way, in my opinion, to format imports (and data structures for that matter), as results in this:
```py
from some.very.very.very.very.long.module.path import (a,
                                                       b,
                                                       c)
```

Reply:
The docs and example are different in order that they may be more compact. While what you show is technically a possibility, I have not actually seen what you describe ever be a problem in actual practice in our specific repo. 

cc @tcmetzger do you have any thoughts about import formatting in docs code examples? The available isort multi line output options are described here: https://pycqa.github.io/isort/docs/configuration/multi_line_output_modes/